scripts/ci/ci-run-sqllogic-tests-cluster.sh

```diff
@@ -21,4 +21,4 @@ fi
 echo "Run suites using argument: $RUN_DIR"
 
 echo "Starting databend-sqllogic tests"
-target/${BUILD_PROFILE}/databend-sqllogictests --handlers ${TEST_HANDLERS} ${RUN_DIR} --enable_sandbox --parallel 8 --skip_file tpcds_q64.test
+target/${BUILD_PROFILE}/databend-sqllogictests --handlers ${TEST_HANDLERS} ${RUN_DIR} --enable_sandbox --parallel 8 --skip_file tpcds_q64.test,tpcds_join_order.test
```

Comment:
why skip the test file?

Reply:
The explain result may be different between standalone mode and cluster mode. I think we don't need to prepare a whole tpcds test suite for it.